index.html

```diff
@@ -14,19 +14,25 @@
     <header>
         <section class="portfolio-header">
             <a class="logo">My Logo</a>
-            <img src="images/Enabled.png" alt="hamburger">
+            <nav class="header-menu">
+                <img src="images/Enabled.png" alt="hamburger" class="hamburger">
+                <ul>
+                   <a href="#"><li>Portfolio</li></a> 
+                   <a href="#"><li>About</li></a> 
+                   <a href="#"><li>Contact</li></a>
+                </ul>
+            </nav>
         </section>
     </header>
-  
     
     <section class="container">
-            <h1 class="salutation">I’m Austin. <br> Glad to see you!</h1>
+            <h1 class=" container-item salutation first">I’m Austin. <br> Glad to see you!</h1>
 
-            <p class="developer"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <p class="container-item developer"> I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
 
-    <h2 class="connect">LET'S CONNECT</h2>
+    <h2 class="container-item connect">LET'S CONNECT</h2>
 
-        <section class="social">
```

Comment:
- [ ] As requested by the previous reviewer your background image should match the Figma design. Please position it on the right place and make sure that every image is present.
- [ ] [Optional] I would suggest you please adjust the spacing for header and content and also make sure to decrese the space between header and section tag. See image below for reference
### Your design
![his](https://github.com/ausiemass/portfolio-austin/assets/81727988/1285d92a-58e0-4e02-a392-b1094bf85d40)
### Figma design
![fimga](https://github.com/ausiemass/portfolio-austin/assets/81727988/1796f0dd-17c0-41f0-a295-dfd6a71b83b7)